packages/ui-components/src/components/Chart/Table/StyledTable.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import styled from 'styled-components';
+import MuiTable from '@material-ui/core/Table';
+
+const DARK_THEME_BORDER = 'rgb(82, 82, 88)';
+
+export const StyledTable = styled(MuiTable)`
```

Comment:
@reviewer This StyledTable is duplicate code of the components/Map/Table/StyledTable.js. I kept them seperate because:
1. `components/Chart` and `components/Map` are bundled in to different bundles so the only way to share them would to put it into the main ui-components bundle
2. The styles are quite specific to the table flippa and I was concerned about creating confusion and maintenance issues having another table in ui-components

In saying that, I'm still not sure about it and happy to be convinced of a way to combine them.